cmd/forest/main.go

```diff
@@ -122,6 +124,9 @@ func createIdentity(args []string) error {
 		usage()
 		return err
 	}
+	if newline.MatchString(name) {
```

Comment:
```suggestion
	if strings.Contains(name, "\n") {
```

Reply:
@aThorp96 I'm still learning how to use the suggested changes feature, sorry that other ones overlapped and couldn't all by applied at once.

Reply:
Its all good haha. I think that was on me for applying them out of order